### ISSUE TITLE
Bug: hreflang default must be present as well

### ISSUE THREAD

**Issue by kodeine** [statamic/seo-pro#44]
as the title says, default hreflang tag should be present along with the other languages.

**Comment by pmeissner** [comment]
```hreflang="x-default"``` should be in there, and I believe the page should include a reference to itself. Currently seo-pro looks like it only outputs the alternates of the other available languages. ref: [Moz](https://moz.com/learn/seo/hreflang-tag)

**Comment by pmeissner** [comment]
Sorry, my comment may be a dupe of this: [https://github.com/statamic/seo-pro/issues/49](issue 49)

**Comment by jackmcdade** [comment]
👍 Closing as duplicate.

**Comment by clementmas** [comment]
This is not a duplicate. The other issue doesn't mention the missing `x-default` tag.

I would have expected SEO Pro to handle this. Instead, I have to add this to my layouts:

```antlers
{{ seo_pro:meta }}

{{ locales:en }}
    <link rel="alternate" href="{{ permalink }}" hreflang="x-default" />
{{ /locales:en }}
```

There should be a `default` option in the sites config or the first one should be picked as the `x-default`.

**Comment by duncanmcclean** [comment]
This issue has been closed for over 4.5 years and was regarding an older version of SEO Pro. If this is still an issue for you, please open a new issue.
